### PR TITLE
Extend Thamos configuration file with additional options

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/common/pull/1072/
Related: https://github.com/thoth-station/user-api/pull/1212
Fixes: https://github.com/thoth-station/thamos/issues/656

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thamos/lib.py [hunk 1]
```diff
@@ -48,7 +48,6 @@
 from .swagger_client import BuildAnalysisApi
 from .swagger_client import Configuration
 from .swagger_client import PythonStack
-from .swagger_client import RuntimeEnvironment
 from .swagger_client import AdviseInput
```
Comment (pacospace) [inline thamos/lib.py:51]: do we remove it because we check from common config now?

**Review by pacospace — COMMENTED** [review]
one question otherwise lgtm :)

File: thamos/lib.py (hunk 1 above)
Comment (fridex) [reply thamos/lib.py:51]: We don't want to have fixed runtime environment entries as provided by the API model. In that case, we will introduce a tight coupling between thamos and user-api versions. Users will need to make sure thamos version is the right for communicating with thoth backend (exact match), which is not nice. With this change, the API schema is not strictly enforced, but users will get warning messages if they use an older or newer thamos version when communicating with thoth (can be seen logs).

Comment (pacospace) [reply thamos/lib.py:51]: Thanks for exaplanation :)

**Review by pacospace — APPROVED** [review]
💯 🍺

**Pull request merged**